wallet/src/de/schildbach/wallet/ui/main/MainViewModel.kt

```diff
@@ -127,6 +127,7 @@ class MainViewModel @Inject constructor(
                     .debounce(THROTTLE_DURATION)
                     .onEach { refreshTransactions(filter) }
             }
+            .catch { analytics.logError(it, "is wallet null: ${walletData.wallet == null}") }
```

Comment:
There is some amount of NRE in this observer: https://console.firebase.google.com/u/1/project/dash-wallet-firebase/crashlytics/app/android:hashengineering.darkcoin.wallet/issues/dfe982a7e3de71274bd2b43712eb892b?time=last-ninety-days&types=crash&sessionEventKey=6317BF81017E00014989D2CC4740508C_1718831905515279144

Seems related to the fact that the `wallet` is null sometimes.

Reply:
The `null` `wallet` is an elusive error mostly seen on those Krip phones.